### PR TITLE
[WOR-1558] Remove unnecessary workspace-dashboard integration test

### PR DESCRIPTION
**Pull request by cahrens**
### Jira Ticket: https://broadworkbench.atlassian.net/browse/WOR-1558

This functionality is now covered by unit tests (with the exception of color contrast testing). The test, which predated workspace dashboard unit tests, was originally much longer. As unit tests were added, most of it was stripped away. What remained was largely already covered, so deleting the entire test makes sense.

### CODE REVIEW

File: integration-tests/tests/workspace-dashboard.js [hunk 1]
```diff
@@ -1,80 +0,0 @@
...
-
-const testGoogleWorkspace = _.flow(
-  withWorkspace,
-  withUserToken
-)(async ({ page, token, testUrl, billingProject, workspaceName }) => {
-  await gotoPage(page, testUrl);
-  await setGcpAjaxMockValues(page, billingProject, workspaceName);
-  const dashboard = workspaceDashboardPage(page, token, workspaceName);
-  await dashboard.visit();
-  await dashboard.assertDescription('About the workspace');
```
Comment (cahrens) [inline integration-tests/tests/workspace-dashboard.js:62]: Covered by [WorkspaceDashboard.test.ts](https://github.com/DataBiosphere/terra-ui/blob/a88fa5e64fb22d56cf40b35c6821314c53515a26/src/workspaces/dashboard/WorkspaceDashboard.test.tsx#L34)

File: integration-tests/tests/workspace-dashboard.js [hunk 2]
```diff
@@ -1,80 +0,0 @@
...
-
-  // Check selected items in cloud information
-  const currentDate = new Date().toLocaleDateString();
-  await dashboard.assertCloudInformation([`Bucket SizeUpdated on ${currentDate}0 B`]);
```
Comment (cahrens) [inline integration-tests/tests/workspace-dashboard.js:66]: Coverage added to CloudInformation.test.ts

File: integration-tests/tests/workspace-dashboard.js [hunk 3]
```diff
@@ -1,80 +0,0 @@
...
-  // Different versions have different ways of dealing with the SVG text.
-  await page.waitForXPath('//*[contains(normalize-space(.),"Cloud NameGoogle Cloud Platform") or @title="Google Cloud Platform"]');
```
Comment (cahrens) [inline integration-tests/tests/workspace-dashboard.js:68]: Covered by [CloudInformation.test.ts](https://github.com/DataBiosphere/terra-ui/blob/a88fa5e64fb22d56cf40b35c6821314c53515a26/src/workspaces/dashboard/CloudInformation.test.ts#L88)

File: integration-tests/tests/workspace-dashboard.js [hunk 4]
```diff
@@ -1,80 +0,0 @@
...
-
-  // Verify expected tabs are present.
-  await dashboard.assertTabs(['data', 'analyses', 'workflows', 'job history'], true);
```
Comment (cahrens) [inline integration-tests/tests/workspace-dashboard.js:71]: Covered by [WorkspaceTabs.test.ts.](https://github.com/DataBiosphere/terra-ui/blob/a88fa5e64fb22d56cf40b35c6821314c53515a26/src/workspaces/container/WorkspaceTabs.test.ts#L85)

File: integration-tests/tests/workspace-dashboard.js [hunk 5]
```diff
@@ -1,80 +0,0 @@
...
-
-  // Check accessibility.
-  await verifyAccessibility(page);
```
Comment (cahrens) [inline integration-tests/tests/workspace-dashboard.js:74]: Various components on the workspace dashboard test their accessibility in unit tests. We do lose color contrast testing though.

**Review by okotsopoulos — APPROVED** [review]
That's great!

**Review by aherbst-broad — APPROVED** [review]

**Pull request merged**